index/scorch/mergeplan/merge_plan.go

```diff
@@ -37,6 +37,12 @@ type Segment interface {
 	// Size of the live data of the segment; i.e., FullSize() minus
 	// any logical deletions.
 	LiveSize() int64
+
+	HasVector() bool
+
+	FullVectorsByteSize() uint64
```

Comment:
Better naming, perhaps `IndexedVectorsBytes()`

Also looking at the code below, it seems LiveVectorsBytesSize() is the only needed exported API.

Reply:
Done